### ISSUE TITLE
v-select doesn't work with screen reader

### ISSUE THREAD

**Issue by kodeo** [vuetifyjs/vuetify#15087]
On tabbing on to the following element using Windows screen reader NVDA...

```
<v-select
  label="Delivery country"
  :items="countryOptions"
  v-model="deliveryCountry"
/>

data: () => ({
  countryOptions: ['UK', 'USA'],
  deliveryCountry: 'UK'
})
```

...the screen reader says "Delivery country UK menu button collapsed submenu, delivery country edit readonly blank". On pressing space to edit the field then using cursor keys to move through the items, the reader says "blank" for each item. On pressing enter to select an item the reader says nothing at all to reflect the new value.

Contrast this with the equivalent `<select>`, where the reader says "Delivery county combo box UK collapsed" initially, reads the values out as you cursor through, e.g. "UK 1 of 2", "USA 2 of 2", and on selection announces the new value "Delivery county combo box USA collapsed".
